_posts/0000-01-02-cloverleaf22.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Find someone who loves you, cares for you, admires you, supports you, makes you happy, instead of looking for the one you love.